### PR TITLE
Added Apache license notes

### PR DESCRIPTION
**Pull request by sveinungkb**
**PS: Public repo & discussion.**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,3 +1,5 @@
+Copyright (C) 2015 Button, Inc
```
Comment (wessmith) [inline LICENSE]: It's currently 2016

Comment (sveinungkb) [inline LICENSE]: Copyright year is from when it started applying, no?

File: LICENSE [hunk 2]
```diff
@@ -7,3 +9,19 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+------------------------------------------------------------------------------
```
Comment (wessmith) [inline LICENSE]: You might want to add an acknowledgements note here and head the license note with the name of the project so it doesn't look like The Android Open Source Project also claims copyright on our SDK

> Acknowledgments
> Portions of this Button Software may utilize the following copyrighted material, the use of which is hereby acknowledged.
> 
> DiskLruCache

File: LICENSE (hunk 1 above)
Comment (wessmith) [inline LICENSE]: I did 2014-2016 as that's what I see in other license files (although for android it would be 2015-2016)

File: LICENSE (hunk 2 above)
Comment (sveinungkb) [inline LICENSE]: See ec49a86

File: LICENSE (hunk 1 above)
Comment (mikeybtn) [inline LICENSE]: can we make this `LICENSE.txt` or `LICENSE.md`?

File: LICENSE [hunk 3]
```diff
@@ -7,3 +9,25 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
+
+-------------------------ACKNOWLEDGEMENTS--------------------------------------
```
Comment (mikeybtn) [inline LICENSE]: Let's just enumerate them by class, leaves little unnecesasry room for interpretation:

```
Third-party Licenses

## com.whatever.classpath.to.DiskLruCache

<full license statement>

## com.whatever.classpath.to.PagerTabStrip

<full license statement>

## com.whatever.classpath.to.PagerTitleStrip

<full license statement>

```

Comment (sveinungkb) [inline LICENSE]: Done in e3172e7bfb7689b59cf1156d552e87f863cd2a41

File: LICENSE.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+-------------------------ACKNOWLEDGEMENTS--------------------------------------
```
Comment (mikeybtn) [inline LICENSE.md]: Replace line:

```
## Third-party components
```

File: LICENSE.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+
+Portions of this Button Software may utilize the following copyrighted material,
```
Comment (mikeybtn) [inline LICENSE.md]: Strike this paragraph.

**Pull request merged**